skt/reporter.py

```diff
@@ -273,29 +273,26 @@ def stateconfigdata(self, mergedata):
         return mergedata
 
     def update_mergedata(self):
+        """Get information about the patch(es) being merged.
+
+        Returns: A dictionary of information about the merge, including the
```

Comment:
Google Python Style Guide says the "Returns" description should start on the next line. It would be good to keep the code consistent. This fix is not required for my ACK, though.

Reply:
Fixing.